app/eml_parser.py

```
@@ -0,0 +1,415 @@
+            body_features = self._extract_body_content(msg)
+            features.update(body_features)
+            
+            # Extract security indicators
+            security_features = self._extract_security_features(msg)
+            features.update(security_features)
+            
+            # Extract hidden/embedded content
+            hidden_features = self._extract_hidden_content(msg)
+            features.update(hidden_features)
+            
+            return features
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)